.gitignore

```diff
@@ -6,6 +6,7 @@ rice-box.go
 /filebrowser
 /filebrowser.exe
 /dist
+/vendor
```

Comment:
This project is not using vendoring and this config is redundant

Reply:
<img width="314" alt="Screenshot 2025-03-20 at 1 15 54" src="https://github.com/user-attachments/assets/26fd117d-45a8-46d4-abd7-b273060dc976" />

How can I debug without running `go mod vendor`?